tilt/README.md

```diff
@@ -0,0 +1,37 @@
+# Using Tilt (tilt.dev) with the Portainer Agent
+
+## Directory Contents
+This directory contains some tiltfiles for use with Tilt.
+
+## Getting started for developers
```

Comment:
does it allow debug? putting breakpoints?

Reply:
It's completely possible.  But I didn't get around to doing that.  It requires attaching delv. 

Details here:
https://www.kriskelly.me/go-dlv-tilt-debugging/